src/index.html

```diff
@@ -40,19 +40,20 @@
       <!-- Hero -->
```

Comment:
- [ OPTIONAL] Always wrap your logo in an `<a>` tag to make it clickable for semantic and conventional reasons, and also to improve the user experience.